python_src/src/lamp_py/migrations/versions/performance_manager/be4c10c548f0_initial_rds_schema.py

```diff
@@ -197,11 +197,42 @@ def upgrade() -> None:
         ["trip_stop_hash"],
         unique=True,
     )
+    create_update_on_function = """
+        CREATE OR REPLACE FUNCTION update_modified_columns() 
+        RETURNS TRIGGER AS $$ 
+        BEGIN 
+        NEW.updated_on = CURRENT_TIMESTAMP; 
+        RETURN NEW;
+        END; $$ LANGUAGE plpgsql
+    """
+    op.execute(create_update_on_function)
+
+    for table in ["vehicle_events", "vehicle_trips", "vehicle_event_metrics"]:
+        trigger_name = f"update_{table}_modified".lower()
+        create_trigger = """
+            CREATE TRIGGER %s BEFORE UPDATE ON "%s" 
+            FOR EACH ROW EXECUTE PROCEDURE update_modified_columns();
+        """ % (
+            trigger_name,
+            table,
+        )
+        op.execute(create_trigger)
+
```

Comment:
Add control of trigger/function creation/destruction to initial alembic migration file.